lib/constants.js

```diff
@@ -31,9 +31,9 @@ import { contractVersions, prizePoolBuilders } from '@pooltogether/current-pool-
 
 export const SENTINEL_ADDRESS = '0x0000000000000000000000000000000000000001'
 
-export const SUPPORTED_NETWORKS = [1, 4, 42, 77, 100, 137, 31337, 80001]
+export const SUPPORTED_NETWORKS = [1, 4, 42, 56, 77, 99, 100, 137, 31337, 80001]
```

Comment:
Do we want 99 `POA Network Core` or 97 `Binance Smart Chain Testnet`

Reply:
Ah I see it in https://github.com/pooltogether/pooltogether-reference-pool-ui/pull/59